core-api/src/main/java/com/redhat/lightblue/Response.java

```diff
@@ -18,26 +18,40 @@
  */
 package com.redhat.lightblue;
 
+import java.util.Iterator;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.node.ArrayNode;
 import com.fasterxml.jackson.databind.node.JsonNodeFactory;
 import com.fasterxml.jackson.databind.node.ObjectNode;
-
-import java.util.ArrayList;
-import java.util.Iterator;
-import java.util.List;
+import com.redhat.lightblue.util.Error;
+import com.redhat.lightblue.util.JsonUtils;
 
 /**
  * Response information from mediator APIs
  */
 public class Response extends BaseResponse  {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(Response.class);
+
     private static final String PROPERTY_PROCESSED = "processed";
     private static final String PROPERTY_RESULT_METADATA = "resultMetadata";
 
     private JsonNode entityData;
     private List<ResultMetadata> resultMetadata;
 
+    private int responseDataSizeB = 0;
+    private int maxResultSetSizeB = -1, warnResultSetSizeB = -1;
+    private Request forRequest;
+    private boolean warnThresholdBreached = false;
+
+    // Response has no access to CrudConstants
```

Comment:
Can we make the error constants more visible?

If it's high effort for some reason, maybe not worth it right now.